gtnet/run.py

```diff
@@ -19,3 +20,24 @@ def predict(fasta_path, domain, vocab, **kwargs):
                                   pad_value=8)
         # 2. pass chunks into model
         output = model.run(None, {input_name: bidir_seq.astype(np.int64)})[0]
+
+
+def predict(argv=None):
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
log to standard error

Reply:
You can set up a logger to write to a specific stream, like stderr or a specific file.

Reply:
https://github.com/exabiome/deep-taxon/blob/master/src/exabiome/utils.py#L36-L46